src/modules/user/user.service.ts

```diff
@@ -13,4 +13,8 @@ export class UserService {
   async findAll(): Promise<UserEntity[]> {
     return this.userRepository.find();
   }
+
+  async putUser(user_: UserEntity): Promise<any> {
```

Comment:
Put normalmente utilizamos para quando vamos atualizar dados já existentes

```suggestion
  async createUser(user_: UserEntity): Promise<any> {
```